e2e-tests/playwright/package.json

```diff
@@ -12,29 +12,29 @@
         "playwright-ui": "cross-env NODE_OPTIONS='--no-experimental-fetch' playwright test --ui",
         "test-slomo": "cross-env NODE_OPTIONS='--no-experimental-fetch' PW_SNAPSHOT_ENABLE=true PW_SLOWMO=1000 playwright test",
         "show-report": "npx playwright show-report",
-        "postinstall": "npx playwright install"
+        "playwright-install": "npx playwright install"
```

Comment:
Does that mean this will not run automatically after npm install?

Reply:
Yes. In local dev, it sometimes happened as well, that's why I added previously the need to install it manually - https://github.com/mattermost/mattermost/blob/master/e2e-tests/playwright/README.md#2-install-dependencies-and-run-the-test.
This is done so it does not need to remove and re-install the browsers when using the Playwright docker image in CI but if it's a pain in local dev, I can switch `postinstall` using script and add a flag to be able to skip in CI.

Reply:
Updated so it will remain auto-install in local and have the ability to skip in CI - https://github.com/mattermost/mattermost/pull/26464/commits/fa4aba5cdd39258f49ce091a30c3a784b3976b50

Also, please take a look at the update in config types - https://github.com/mattermost/mattermost/pull/26464/commits/2ba4a81c9041625550a6b922b67ea9f1705d5555

Reply:
thanks for the update, yes i believe it would be helpful for the contributors one less step for them to do.

Skipping in CI and updated config LGTM